features/steps/shell.py

```diff
@@ -64,7 +64,17 @@ def when_i_run_command(
 ):
     command = process_template_vars(context, command)
     prefix = get_command_prefix_for_user_spec(user_spec)
-    full_cmd = prefix + shlex.split(command)
+    if "pro" in command.split():
+        command = command.replace(
+            "pro",
+            "python3 -m coverage run \
+                --source=/usr/lib/python3/dist-packages/uaclient /usr/bin/pro",
+            1,
+        )
+
+    split_command = shlex.split(command)
+    print(split_command)
+    full_cmd = prefix + split_command
```

Comment:
If this is working, we can consolidate this back into one line
```suggestion
    full_cmd = prefix + shlex.split(command)
```

Reply:
This print statement slipped through, should be working so will revert to one line